__tests__/components/login.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { shallow, mount } from 'enzyme';
+import store from '../../src/redux/store';
+import LoginComponent, { Login } from '../../src/components/pages/Login';
+
+const props = {
+  ui: {
+    loading: false,
+  },
+  login: jest.fn(),
+};
+
+let login;
+
+describe('Login Component...', () => {
+  beforeAll(() => {
+    login = shallow(<Login {...props} />);
+    login.setProps({ ...props, ui: { ...props.ui, loading: true } });
+  });
+  it('Should render login component', () => {
+    const component = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <LoginComponent />
+        </MemoryRouter>
+      </Provider>,
+    );
+    expect(component).toHaveLength(1);
+  });
+
+  it('Should render login component', () => {
+    expect(login).toMatchSnapshot();
+  });
+  it('Should find one form', () => {
+    const form = login.find('form');
+    expect(form.length).toBe(1);
+  });
+  it('Should give initial state', () => {
+    expect(login.state()).toBeDefined();
+  });
+
+  describe('Input tests...', () => {
+    it('should type in the Email', () => {
+      const Email = login.find('input[name="email"]');
+      Email.simulate('change', {
+        target: { value: 'danimahoro75@gmail.com' },
+      });
+      expect(login.state('email')).toEqual('danimahoro75@gmail.com');
+    });
+    it('should type in the password', () => {
+      const password = login.find('input[name="password"]');
+      password.simulate('change', {
+        target: { value: 'Esther@123!' },
+      });
+      expect(login.state('password')).toEqual('Esther@123!');
+    });
+  });
+  describe('submit button test...', () => {
+    let instance;
+    let submitButton;
+    beforeAll(() => {
+      instance = login.instance();
+      submitButton = login.find('input[type="submit"]');
+      submitButton.simulate('click');
+    });
+    it('should make a request to the server', () => {
```

Comment:
This test doesn't appear to have an assertion, there is no `expect` statement in it. Would you kindly explain what you are testing for here?